src/events/guildMemberAdd.js

```diff
@@ -49,8 +49,7 @@ bot.on('guildMemberAdd', async function (guild, member) {
         inline: true
     }, {
         name: 'Created',
-        value: moment(member.user.createdAt).tz('Etc/GMT').format('llll') +
-            ` GMT\n(${moment.duration(-1 * (moment() - moment(member.user.createdAt))).humanize(true)})`,
+        value: `<t:${moment(member.user.createdAt).unix()}> (<t:${moment(member.user.createdAt).unix()}:R>)`, // we need to show there account age too right.

```

Comment:
I know the code was already like that but is it possible to improve it by creating a variable to contain the value of `moment(member.user.createdAt).unix()` and call this variable in the string interpolation please?

Reply:
Done